framework/Optimizers/RavenSampled.py

```diff
@@ -332,7 +332,12 @@ def finalizeSampler(self, failedRuns):
         bestTraj = traj
         bestValue = val
     # further check active unfinished trajectories
-    traj = 0
+    ## FIXME why should there be any active, unfinished trajectories when we're cleaning up sampler?
+    traj = 0 # FIXME why only 0?? what if it's other trajectories that are active and unfinished?
```

Comment:
The two FIXME notes here aren't part of this PR; they're comments on previously-added code, FYI.